### PR TITLE
Change to Foundation css for home page and widgets

### PR DESCRIPTION
**Pull request by jdungan**

### CODE REVIEW

**Comment by jdungan** [comment]
This PR is ready for review.

File: codesy/templates/base.html [hunk 1]
```diff
@@ -1,62 +1,67 @@
...
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-      <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
-
-    <link rel="stylesheet" href="{% static "css/styles.css" %}">
+    <link rel="stylesheet" href="https://dhbhdrzi4tiry.cloudfront.net/cdn/sites/foundation.min.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/foundation-icons/3.0/foundation-icons.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/motion-ui/1.1.1/motion-ui.min.css" />
```
Comment (jdungan) [inline codesy/templates/base.html]: Gave into sexy animations.  Could be talked out of it...

File: codesy/templates/base.html [hunk 2]
```diff
@@ -1,62 +1,67 @@
...
+                 />
+           </a> codesy.io llc; Git Logo <a rel="license" href="http://creativecommons.org/licenses/by/3.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by/3.0/80x15.png" /></a> <a href="http://www.jasonlong.me/">Jason Long</a>; Icons by
+           <a href="http://zurb.com/playground/foundation-icon-fonts-3/">Foundation</a>
+        </p>
       </footer>
     </div>
 
     <script src="//cdn.jsdelivr.net/jquery/2.1.3/jquery.min.js"></script>
-    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/foundation/6.2.0/foundation.min.js"></script>
```
Comment (jdungan) [inline codesy/templates/base.html]: We only use this for closing message ... for now.

File: api/templates/includes/vote_form.html [hunk 3]
```diff
@@ -1,9 +1,20 @@
...
+<div class="callout warning">
+   <h3>You haven't voted yet. Do you approve this claim?.</h3>
+   <form id="codesy_vote"
+      class="ajaxSubmit"
+      action="{% url 'vote-list' %}"
+      data-method="POST" >
+      {% csrf_token %}
+      <input name = "claim"  type="hidden" value="{{claim.id}}"></iinput>
+      <div class="switch large">
+         <input class= "switch-input" id="approved" type="checkbox" >
```
Comment (groovecoder) [inline api/templates/includes/vote_form.html]: I think you just need `name="approved"` here to pass the name parameter in the HTTP post.

File: codesy/urls.py [hunk 4]
```diff
@@ -12,12 +12,10 @@
     '',
     # Static home/ explanation pages
     url(r'^$', views.Home.as_view(), name='home'),
-    url(r'^how-it-works$',
-        TemplateView.as_view(template_name="how_it_works.html"),
-        name='how_it_works'),
-    url(r'^faq$', TemplateView.as_view(template_name="faq.html"), name='faq'),
```
Comment (groovecoder) [inline codesy/urls.py]: giphy

File: static/js/home-app.js [hunk 5]
```diff
@@ -54,9 +57,11 @@ $(window).load(function () {
       number: $('#cc-number').val(),
       exp_month: $('#cc-ex-month').val(),
       exp_year: $('#cc-ex-year').val(),
-      cvc: $('#ex-cvv').val()
+      cvc: $('#cvc').val()
     };
 
+    handleResponse = new stripeResponse($('form [name="csrfmiddlewaretoken"]').val())
```
Comment (groovecoder) [inline static/js/home-app.js]: kungfury

**Pull request merged**